tests/platform_tests/test_intf_fec.py

```diff
@@ -99,12 +110,10 @@ def test_config_fec_oper_mode(duthosts, enum_rand_one_per_hwsku_frontend_hostnam
         config_status = duthost.command("sudo config interface fec {} {}"
                                         .format(intf['interface'], fec_mode))
         if config_status:
-            pytest_assert(wait_until(30, 2, 0, duthost.is_interface_status_up, intf["interface"]),
-                          "Interface {} did not come up after configuring FEC mode".format(intf["interface"]))
-            # Verify the FEC operational mode is restored
-            post_fec = get_fec_oper_mode(duthost, intf['interface'])
-            if not (post_fec == fec_mode):
-                pytest.fail("FEC status is not restored for interface {}".format(intf['interface']))
+            # interfaces that are not copper could take longer to configure fec
```

Comment:
@nhe-NV Why FEC behavior is different between copper and active cables? Any IEEE spec?

Reply:
Hi @prgeor  This is not related to the fec itself, it it due to the opticial module will take longer to come up than the copper, when run ""sudo config interface fec rs", in sonic, it will first do port shutdown, then change fec mode, and startup port, the startup port will take longer for optical

Reply:
@nhe-NV not clear. you mean the datapath of the CMIS module needs time to initialize when we change FEC? Who is shutting the port down when you say "do port shutdown"?

Reply:
Hi @prgeor  It is not related to the fec mode change itself, , when the port is in down status, if we want to make the port come up, we can do "config interface Ethernet0 startup", if the module it optical, then it will take longer for us to wait the ports comeup. 
And when we try to change the fec mode with the "sudo config interface fec rs", internally, it will first shutdown the port, then change fec mode, and starup port. so we can assume "sudo config interface fec rs" = "config interface Ethernet0 shutdown" + " config interface fec rs"  + "config interface Ethernet0 startup". since the starup up will take longer on the optical module, so we also need to wait longer to check if the fec mode is changed or not for the optical module.